lib/pages/pattern_list.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:MobileOne/pages/create_list.dart';
+
+Widget patternLists(BuildContext context) {
```

Comment:
What is this file ? Is that a widget ? Is that a function ? Is that anything else ?

If you want to create a specific widget just create a Widget. Not a function that will generate a Widget